abb_irb1600_6_145_support/package.xml

```diff
@@ -0,0 +1,40 @@
+<package>
+  <name>abb_irb1600_6_145_support</name>
+  <version>1.0.5</version>
+  <description>
+    <p>
+      ROS-Industrial support for the ABB IRB 1600 - 6/1.45.
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for ABB IRB 1600 - 6/1.45 manipulators. This currently includes the base model.
+    </p>
+    <p>
+      Joint limits and max joint velocities are based on the information in
+      the ABB data sheets.  All URDFs / XACROs are based on the
+      default motion and joint velocity limits.
+    </p>
+    <p>
+      Before using any of the configuration files and / or meshes included
+      in this package, be sure to check they are correct for the particular
+      robot model and configuration you intend to use them with.
+    </p>
+  </description>
+  <author email="aabdelmawgoud@swri.org">Ahmed Abdelmawgoud (Southwest Research Institute)</author>
+  <maintainer email="larmstrong@swri.org">Levi Armstrong (Southwest Research Institute)</maintainer>
```

Comment:
This maintainer might be outdated, FYI [the latest upstream](https://github.com/ros-industrial/abb_experimental/blob/565ee52000276945eb3cf1c60eaa5cf85dfc1691/abb_experimental/package.xml#L7).

Reply:
Nevermind my comment for this line. Removing.